iroh/src/commands.rs

```diff
@@ -567,6 +567,27 @@ pub enum BlobCommands {
     /// Delete content on the node.
     #[clap(subcommand)]
     Delete(self::delete::Commands),
+    /// Get a ticket to share this blob.
+    Share {
+        /// Hash of the blob to share.
+        hash: Hash,
+        /// Add this token to the ticket.
+        #[clap(long)]
+        token: Option<String>,
+        /// Do not include derp reion information in the ticket.
```

Comment:
```suggestion
        /// Do not include DERP region information in the ticket. (advanced)
```

Reply:
Lol. Want me to hide it?

Reply:
I was tempted to ask for it to be hidden, but I think it might be useful for working with users, while we debug stuff, and I'd rather not have other engineers have to fumble through source to remember the flag name

Reply:
done in [f4cbea3](https://github.com/n0-computer/iroh/pull/1746/commits/f4cbea3492b16b9de1bb4a9fe95ef7b181aedbf6)